liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppsMojo.java

```diff
@@ -111,72 +121,81 @@ private void installDependencies() throws Exception {
             }
         }
     }
-    
-    protected void installProject() throws Exception {
-        if (isSupportedType(project.getPackaging())) {
+
+    protected void installProject(boolean thin) throws Exception {
+        String packagingType = project.getPackaging();
+        if (isSupportedType(packagingType)) {
             if (looseApplication) {
-                installLooseApplication(project);
+                installLooseApplication(project, thin);
             } else {
-                installApp(project.getArtifact());
+                installApp(project.getArtifact(), thin);
             }
         } else {
             throw new MojoExecutionException(
                     MessageFormat.format(messages.getString("error.application.not.supported"), project.getId()));
         }
     }
 
-    private void installLooseApplication(MavenProject proj) throws Exception {
+    private void installLooseApplication(MavenProject proj, boolean thin) throws Exception {
         String looseConfigFileName = getLooseConfigFileName(proj);
         String application = looseConfigFileName.substring(0, looseConfigFileName.length() - 4);
         File destDir = new File(serverDirectory, getAppsDirectory());
         File looseConfigFile = new File(destDir, looseConfigFileName);
         LooseConfigData config = new LooseConfigData();
         switch (proj.getPackaging()) {
-            case "war":
-                validateAppConfig(application, proj.getArtifactId());
+        case "war":
+            if (thin) {
+                log.info(MessageFormat.format(messages.getString("info.loose.application.not.supported"),
+                        proj.getPackaging()));
+                installApp(proj.getArtifact(), thin);
```

Comment:
So in this case, the code goes ahead and installs it as a non-loose application?

Reply:
Yes, similar to the default case.